### ISSUE TITLE
module github.com/golang/protobuf is deprecated: Use the "google.golang.org/protobuf" module instead

### ISSUE THREAD

**Issue by leiyiz** [kubernetes-csi/csi-lib-utils#115]

**Comment by A-Hilaly** [comment]
Thought this was going to be easy - but apparently it's a little more work than expected. Will give it a second shot but before need to spend sometime reading the packages and `protosanitizer` package.

/assign

**Comment by AdityaVardhanSingh** [comment]
Hey, I want to work on this but I am new to this so can you help with the resources to get started with the issue?

**Comment by niranjandarshann** [comment]
@leiyiz  The issue is resolved in https://github.com/kubernetes-csi/external-snapshotter/pull/829 PR and merged in main.

**Comment by leiyiz** [comment]
sorry for not being responsive on this - it still looks like an issue looking at [go.mod](https://github.com/kubernetes-csi/csi-lib-utils/blob/master/go.mod#L9).

i suspect it's not straight forward as it appears to be. the priority isn't terribly high tho.

**Comment by leiyiz** [comment]
yeah it's quite complicated actually - a lot of the packages are not found or not compatable in the new protobuf module..

**Comment by huww98** [comment]
github.com/golang/protobuf is moved to indirect now.

**Comment by niranjandarshann** [comment]
Yes agree lots of file is still using this dependencies. mostly in CHANGELOG and vendors folder.

**Comment by niranjandarshann** [comment]
Can we do changes in vendors go file as well?
